src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/OverrideEqualsOnOverloadingOperatorEqualsTests.Fixer.cs

```diff
@@ -137,7 +138,10 @@ public override bool Equals(object obj)
         [Fact]
         public async Task CA2224Async()
         {
-            await VerifyVB.VerifyCodeFixAsync(@"
+            await new VerifyVB.Test
+            {
+                CodeFixTestBehaviors = CodeFixTestBehaviors.SkipLocalDiagnosticCheck,
```

Comment:
Can we file an issue to track fixing all analyzers reporting non-local diagnostics?

Reply:
➡️ #6598